letsencrypt/le_util.py

```diff
@@ -202,6 +203,37 @@ def safely_remove(path):
             raise
 
 
+def get_os_info():
+    """
+    Get Operating System type/distribution and major version
```

Comment:
This doesn't quite work. If nothing else, we need a new line before `:returns:` or the documentation isn't formatted properly.

Additionally, `grep` the project for `rtype: tuple` to see how other functions with similar return values have been documented.